front-end/src/IndividualShow.js

```diff
@@ -109,14 +109,14 @@ const IndividualShow = ({ id }) => {
     // Fetch show meta-information from the API
     axios
       .get(
-        `https://my.api.mockaroo.com/shows/${id}.json?key=${process.env.REACT_APP_MOCKAROO_KEY}`
+        `http://localhost:3000/shows/${id}`
```

Comment:
Looks good! The only thing is we've changed the back-end port to 4000 to avoid the back-end and front-end from running on the same port, so this should be updated to reflect that

Reply:
forgot about that, I changed it now!